wordpress-mobile/WordPress-iOS
# Reader: This rolls back the temporary patch in #6633.

Title has it.  Now that the reader detail is again presented full screen the patch is no longer needed.

To test:
Repeat the steps outlined in #6633 and #6288 and confirm the scroll offset does not jump.

Needs review: @frosty
